src/BlockTypes/ProductQuery.php

```diff
@@ -138,7 +138,7 @@ public function add_navigation_link_directives( $block_content, $block, $instanc
 
 				$navigation_link_payload = array(
 					'prefetch' => $is_previous_or_next,
-					'scroll'   => true,
+					'scroll'   => false,
```

Comment:
Nitpicking: I believe it would enhance the code if we could eliminate the need to explicitly pass `false` when we don't intend to enable the scroll functionality. Perhaps we can modify the condition [here](https://github.com/woocommerce/woocommerce-blocks/blob/ed0decf425667322a3b9b506822f10371cd18b9b/assets/js/interactivity/directives.js#L183) to achieve this. What are your thoughts on this?

Reply:
We would have to decide what behavior we want to have by default. 🤔 Depending on the API we'd take as an inspiration, the `scroll` value would be:

- **true**
   - [`location.assign()`](https://developer.mozilla.org/en-US/docs/Web/API/Location/assign)
   - [`location.replace()`](https://developer.mozilla.org/en-US/docs/Web/API/Location/replace)
   - [`navigation.navigate()`](https://developer.mozilla.org/en-US/docs/Web/API/Navigation/navigate) (experimental 🧪)
- **false**
   - [`history.pushState()`](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState)
   - [`history.replaceState()`](https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState)

I prefer to follow what the future `navigation.navigate()` API is doing, i.e., to let the browser decide (most of the time, it would be scrolling by default). In any case, I think it would be a good idea to set the scroll behavior explicitly, at least for now.

Pinging @luisherranz in case he wants to add anything else. 🙂

Reply:
For full-page client-side navigation, there's no doubt that the default should be _scroll to the top_. 

For _inner-content_ client-side navigation (what we are doing here), I'm not so sure. Maybe defaulting to not scrolling makes more sense.

So good points 🙂

Let's leave the default as `true` for now, but keep an eye on it!